tests/system/providers/google/cloud/bigquery/example_bigquery_queries_async.py

```diff
@@ -0,0 +1,264 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for Google BigQuery service.
+Uses Async version of the Big Query Operators
+
+"""
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.empty import EmptyOperator
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCheckAsyncOperator,
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryGetDataAsyncOperator,
+    BigQueryInsertJobAsyncOperator,
+    BigQueryIntervalCheckAsyncOperator,
+    BigQueryValueCheckAsyncOperator,
+)
+
+PROJECT_ID = os.getenv("SYSTEM_TESTS_GCP_PROJECT", "")
+DATASET_NAME = os.getenv("GCP_BIGQUERY_DATASET_NAME", "test_dataset")
+GCP_CONN_ID = os.getenv("GCP_CONN_ID", "google_cloud_default")
+LOCATION = os.getenv("GCP_LOCATION", "us")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
```

Comment:
To follow new design of system tests, less env vars:
```suggestion
ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")

DAG_ID = "bigquery_queries_async"

DATASET_NAME = f"dataset_{DAG_ID}_{ENV_ID}"
LOCATION = "us"

EXECUTION_TIMEOUT = 6
```
We can drop GCP_CONN_ID argument from the operators

Reply:
Done